libs/cdk/package.json

```diff
@@ -42,7 +42,7 @@
     "url": "https://github.com/rx-angular/rx-angular.git"
   },
   "peerDependencies": {
-    "@angular/core": "^16.0.0",
+    "@angular/core": "^17.0.0",
     "rxjs": "^6.5.3 || ^7.4.0"
```

Comment:
is the rxjs version still valid? We changed some APIs like `connectable` as I remember correctly.

Reply:
Yes, it's still valid, the PR where I changed some APIs is not merged yet (#1638), I'll check about compatibility in the corresponding PR.